lalrpop-test/src/allow_catch_all.lalrpop

```diff
@@ -0,0 +1,16 @@
+grammar;
```

Comment:
It might be good to test that the precedence is correct by running the parser on some different inputs in https://github.com/lalrpop/lalrpop/blob/master/lalrpop-test/src/lib.rs.